Task_DEV-2/Converter.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task_DEV_2
+{
+    // Class which gets string from "Checker" and convert it 
+    // in postfix format
+    class Converter
+    {
+        private string inputString;
+        private string expressionOperator = "*-+/()";
+
+        // concstructor 
+        public Converter(string inputString)
+        {
+            this.inputString = inputString;
+        }
+
+        // splittig string to array of string
+        // in this array elements is operator or variable
+        private string[] SplittingString()
```

Comment:
Can't understand implementation logics, try to decompose and simplify, add comments